audit/projects/kubernetes-public/services/monitoring/dashboards/spiffxp-artifacts.json

```diff
@@ -0,0 +1 @@
+{"displayName":"spiffxp-artifacts","mosaicLayout":{"columns":12,"tiles":[{"height":4,"widget":{"title":"GCS Bucket - Sent bytes for .*k8s-artifacts-prod.* by label.bucket_name [SUM]","xyChart":{"chartOptions":{"mode":"COLOR"},"dataSets":[{"legendTemplate":"${resource.labels.bucket_name}","minAlignmentPeriod":"60s","plotType":"LINE","timeSeriesQuery":{"timeSeriesFilter":{"aggregation":{"alignmentPeriod":"60s","crossSeriesReducer":"REDUCE_SUM","groupByFields":["resource.label.\"bucket_name\""],"perSeriesAligner":"ALIGN_RATE"},"filter":"metric.type=\"storage.googleapis.com/network/sent_bytes_count\" resource.type=\"gcs_bucket\" resource.label.\"bucket_name\"=monitoring.regex.full_match(\".*k8s-artifacts-prod.*\")","secondaryAggregation":{"alignmentPeriod":"60s"}}}}],"timeshiftDuration":"0s","yAxis":{"label":"y1Axis","scale":"LINEAR"}}},"width":6}]},"name":"projects/127754664067/dashboards/9d1e0395-e764-4c42-ae25-d31f43fe157b"}
```

Comment:
This was me manually messing around, I created a useful graph in metrics explorer (for https://github.com/kubernetes/k8s.io/issues/2318#issuecomment-892789246) and wanted someplace to save it.  I personally don't have a problem with these being manually created right now; I'm pretty sure restore would look like manually re-importing from this file.